_pages/speaking.md

```diff
@@ -28,7 +28,7 @@ Need help with your proposal? We've got mentors and helpful tips on our [Speaker
 
 ## Why Speak at DjangoCon US?
 
--   Presenters receive a free ticket to DjangoCon US! (Travel costs are not included, but potential speakers were encouraged to apply for an <a href="/opportunity-grants/">opportunity grant</a>.) {% comment %}Opportunity grant decision notifications have all been sent.{% endcomment %}
+-   Presenters receive a **free ticket** to DjangoCon US and up to **$500 in travel cost reimbursement!**
```

Comment:
We should still include the link to opportunity grants if they need more than $500.

Reply:
Good enough for me